src/Components/test/testassets/BasicTestApp/FormsTest/TypicalValidationComponent.razor

```diff
@@ -35,14 +35,24 @@
     </p>
     <p class="ticket-class">
         Ticket class:
-        <InputSelect @bind-Value="person.TicketClass" size="4">
+        @*We specify 'multiple' here, but it has no effect since we are not binding to an array type.*@
+        <InputSelect @bind-Value="person.TicketClass" size="4" multiple>
             <option>(select)</option>
             <option value="@TicketClass.Economy">Economy class</option>
             <option value="@TicketClass.Premium">Premium class</option>
             <option value="@TicketClass.First">First class</option>
         </InputSelect>
         <span id="selected-ticket-class">@person.TicketClass</span>
     </p>
+    <p class="cities">
+        @*Here, the 'multiple' attribute is inferred because we are binding to an array type.*@
+        <InputSelect @bind-Value="person.SelectedCities">
+            <option value="@City.SanFrancisco">San Francisco</option>
+            <option value="@City.Tokyo">Tokyo</option>
+            <option value="@City.London">London</option>
+            <option value="@City.Madrid">Madrid</option>
```

Comment:
Might be good to add a couple slightly more hostile test values to be sure the end-to-end transport works in both directions, e.g., value that contains JSON-sensitive characters like `"`, empty string, and if it's possible to get a `null` then that too.